src/promptflow/promptflow/_sdk/data/docker/Dockerfile.jinja2

```diff
@@ -10,7 +10,7 @@ FROM docker.io/continuumio/miniconda3:latest
 
 WORKDIR /
 
-COPY ./flow /flow
+COPY ./flow/{{env.python_requirements_txt}} /flow/{{env.python_requirements_txt}}
```

Comment:
shall we do a if check here?

Reply:
Can this sentence pass if target forlder doesn't exist?

Reply:
need to consider:

1. `env` does not have a `python_requirements_txt`
2. the file `python_requirements_txt` refers does not exist

Reply:
Updated.

How about just checking `env.python_requirements_txt` before `COPY`? It is checked in line 24 as well.

Reply:
please take a look on the CI workflow result: https://github.com/microsoft/promptflow/actions/runs/7034536381?pr=1297, seems there are some jinja syntax error; for example: 

![image](https://github.com/microsoft/promptflow/assets/38847871/92597965-0ed3-4f95-8f75-cac0913fcb48)

please take a look

Reply:
fixed. thank you @zhengfeiwang